scala-core/src/main/scala/com/baeldung/scala/exceptionhandling/ValidationErrors.scala

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.scala.exceptionhandling
+
+sealed trait ValidationErrors extends Product with Serializable
+
+object ValidationErrors {
+  final case class IllegalLogin (login: String) extends ValidationErrors
+  final case class IllegalPassword(password: String) extends ValidationErrors
+}
+
```

Comment:
```suggestion
```

Please ignore this one. I'm using it just to trigger CI since I don't have the necessary right to do that